server/src/main/java/io/crate/execution/engine/aggregation/impl/NumericStandardDeviationAggregation.java

```diff
@@ -67,7 +67,7 @@ public int compare(V val1, V val2) {
 
         @Override
         public int fixedSize() {
-            return 56;
+            return NumericVariance.FIXED_SIZE;
```

Comment:
I don't think that any numeric/BigDecimal based type has a fixed size, it depends on the value used.
I think we should fix this dedicated.